src/spiritual/spiritual-gui/gui@wunderbyte.com/concepts/gui.Action.js

```diff
@@ -388,10 +388,14 @@ gui.Action = (function using(confirmed, chained) {
 					 * @param {Window} win Remote window
 					 * @param {String} uri target origin
 					 * @param {String} key Spiritkey of xdomain IframeSpirit (who will relay the action)
+					 * SetTimeout because the bug in IE11. It will show null exception when you remove
+					 * an iframe.
```

Comment:
Can you please add the information about the Web Component Polyfill. Because it only get triggered when that polyfill exist.

Reply:
OK